constants.js

```diff
@@ -140,7 +140,7 @@ export const defaultMaxId = 8192;
 
 export const defaultMusicVolume = 0.35;
 
-export const voicePacksUrl = `https://webaverse.github.io/voicepacks/all_packs.json`;
+export const voicePacksUrl = `http://localhost/voicepacks/all_packs.json`;
```

Comment:
If ready for review, pls turn off PR's "draft" state, and clean these testing codes.